src/main/java/net/neoforged/neoforge/common/extensions/IBlockExtension.java

```diff
@@ -976,4 +976,29 @@ default PushReaction getPistonPushReaction(BlockState state) {
     default boolean isEmpty(BlockState state) {
         return state.is(Blocks.AIR) || state.is(Blocks.CAVE_AIR) || state.is(Blocks.VOID_AIR);
     }
+
+    enum BubbleColumnDirection {
```

Comment:
draft PR because I need help figuring out where this enum should go. It is needed to have the method be clean and clear about what the result is. (ideally). Unless there's other ideas that could be done

Reply:
Moved enum now to its own file. might be ok?